FailedBonusActivationActivity/handler.ts

```diff
@@ -163,16 +153,22 @@ export function FailedBonusActivationHandler(
           context.log.warn(
             `FailedBonusActivationHandler|WARN|Failed releasing lock: ${err.body}`
```

Comment:
@balanza why it's a warn and not an error here ? are there scenarios where failing to release the lock it's ok? (ie. the lock wasn't created at all and we're trying here to delete something that does not exists?)